src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -5,26 +5,19 @@ sidebar_order: 3
 
 {% include learn-sdk.md platform="javascript" %}
 
-All our JavaScript-related SDKs provide the same API still there are some differences between them which this section of the docs explains.
-
-## Browser Compatibility
-
-Our JavaScript SDK supports all major browsers. In older browsers, error reports may have a degraded level of detail – for example, missing stack trace data or missing source code column numbers.
-
 The table below shows supported browsers:
```

Comment:
This table seems to show the browsers that we test against automatically. Can we state what "degraded level of detail" you'd get in an "older browser"?

This seems like it'd be more useful as a list of the earliest version of browsers that we support at all and a clear asterisk when needed, e.g. "IE 7-10 Promises will require polyfill".

Reply:
Note: This is what we show today -- so not a blocker to replacing current docs.